runtime/src/tasks/massivethreads/tasks-massivethreads.c

```diff
@@ -699,13 +683,15 @@ void chpl_task_setSerial(chpl_bool state) {
 
 // Get pointer to task private data.
 #ifndef CHPL_TASK_GET_PRVDATA_IMPL_DECL
+#if 0
```

Comment:
The commented-out code here looks wrong in any case; it should be getting task-private information. See

https://github.com/chapel-lang/chapel/pull/4074/commits/b80da99f4441ebea473b85e6df398771a74aa4d8#diff-d428c9d1e3eb148d6b27e05c03657e5cR72

for one way to do that.